orderfilter/validate_js.go

```diff
@@ -4,11 +4,17 @@ package orderfilter
 
 import (
 	"errors"
+	"syscall/js"
+	"time"
 
 	"github.com/0xProject/0x-mesh/packages/browser/go/jsutil"
 	"github.com/0xProject/0x-mesh/zeroex"
 )
 
+// sleepTime is used to force processes that block the event loop to give the
+// event loop time to continue. This should be as low as possible.
```

Comment:
nit: This might be slightly more clear.

```suggestion
// JavaScript event loop time to continue. This should be as low as possible.
```